src/addgrid.f

```diff
@@ -78,9 +75,9 @@ subroutine addgrid(cgrib,lcgrib,igds,igdstmpl,igdstmplen,
       logical needext
  
       ierr=0
-!
-!  Check to see if beginning of GRIB message exists
-!
+!>
```

Comment:
Comments in the code do not get doxygen comment blocks. They remain just ordinary code comments. Only the file and the subroutine get a doxygen documentation block. See some of the PRs I put up for examples.